### ISSUE TITLE
Question: How to initialise Mat

### ISSUE THREAD

**Issue by augustocdias** [twistedfall/opencv-rust#53]
How do initialise the Mat with `ones` or `zeros` like in C++ API?

```
Mat::zeros(3, 3, CV_32F);
```

**Comment by twistedfall** [comment]
It turned out that `MatExpr` generation was largely broken, it's fixed now in 0.19.3. You can now use `zeros` method:

```rust
let zero_mat = Mat::zeros(3, 3, f32::typ())?.to_mat()?;
```

**Comment by phunehehe** [comment]
I wonder if there's a way to make `to_mat` easier to discover. Now knowing it, I can see that `MatExpr` implements `MatExprTrait` which has `to_mat`. But when I was looking for it a bit earlier I couldn't find it. In any case, thanks for pointing out. This works!

**Comment by Tramac** [comment]
> It turned out that `MatExpr` generation was largely broken, it's fixed now in 0.19.3. You can now use `zeros` method:
> 
> ```rust
> let zero_mat = Mat::zeros(3, 3, f32::typ())?.to_mat()?;
> ```

let measurement_noise_cov = Mat::ones(2, 2, CV_32F)?.to_mat()?;

<img width="751" alt="image" src="https://user-images.githubusercontent.com/22740819/211732337-386cff77-7144-4397-ae29-6a6aebd512d3.png">

**Comment by twistedfall** [comment]
Please make sure to import prelude:
```rust
use opencv::prelude::*;
```
